lib/facter/login_defs.rb

```diff
@@ -0,0 +1,49 @@
+# Returns the contents of /etc/login.defs as a hash with downcased keys
+#
+Facter.add('login_defs') do
+  confine do
+    File.exist?('/etc/login.defs') && File.readable?('/etc/login.defs')
+  end
+
+  setcode do
+    def read_login_defs
+      File.read('/etc/login.defs')
+    end
+
+    if ['RedHat','CentOS'].include?(Facter.value(:operatingsystem)) &&
+       Facter.value(:operatingsystemmajrelease) < '7'
+        id_min = 500
```

Comment:
Can you cite where you got this number from? I don't think there should be hardcoded defaults in facts.

Reply:
@jeefberkey I've been using EL for a long time :-D.

Ref https://github.com/sindresorhus/default-uid/issues/1#issuecomment-69051381

Basically, 🤷‍♂️

Reply:
Oh I meant if you left it in there, you should document where it comes from. Embedding knowledge, etc etc